src/iuse.cpp

```diff
@@ -768,6 +769,10 @@ cata::optional<int> iuse::antiparasitic( Character *p, item *it, bool, const tri
         p->remove_effect( effect_bloodworms );
         p->add_msg_if_player( _( "Your skin prickles and your veins itch for a few moments." ) );
     }
+    if( p->has_effect( effect_blood_spiders ) ) {
+        p->remove_effect( effect_blood_spiders );
+        p->add_msg_if_player( _( "Your veins becalm and go still in a wave through your body." ) );
```

Comment:
Not sure about "becalm" (usually used only in reference to sailing vessels?), although it sounds nice.

Reply:
It is usually a sailing term but I was specifically trying to avoid language that sounded like you were having a stoppage in your circulation.

Reply:
```suggestion
        p->add_msg_if_player( _( "Your veins relax in a soothing wave going through your body." ) );
```
Perhaps?